### PR TITLE
M3 base template

### PR DESCRIPTION
**Pull request by OwenScott-GDS**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            excludes += '/META-INF/{AL2.0,LGPL2.1}'
+        }
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.core:core-ktx:1.8.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.activity:activity-compose:1.5.1'
```
Comment (kkuan2011) [inline app/build.gradle:69]: nit: alphabetical order for these dependencies

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.Reply"
+        tools:targetApi="31">
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml:29]: 33

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml:22]: You can remove dataExtractionRules and fullBackupContent and their related files

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        android:theme="@style/Theme.Reply"
+        tools:targetApi="31">
+        <activity
+            android:name=".MainActivity"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.Reply">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml:37]: nit: remove blank line

File: app/src/main/java/com/example/reply/ui/theme/Theme.kt [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    secondary = PurpleGrey80,
+    tertiary = Pink80
+)
+
+private val LightColorScheme = lightColorScheme(
+    primary = Purple40,
+    secondary = PurpleGrey40,
+    tertiary = Pink40
+
+    /* Other default colors to override
```
Comment (kkuan2011) [inline app/src/main/java/com/example/reply/ui/theme/Theme.kt:45]: remove unused code

File: app/src/main/java/com/example/reply/ui/theme/Type.kt [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+// Set of Material typography styles to start with
+val Typography = Typography(
+    bodyLarge = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp,
+        lineHeight = 24.sp,
+        letterSpacing = 0.5.sp
+    )
+    /* Other default text styles to override
```
Comment (kkuan2011) [inline app/src/main/java/com/example/reply/ui/theme/Type.kt:34]: same here

File: app/src/main/res/xml/backup_rules.xml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kkuan2011) [inline app/src/main/res/xml/backup_rules.xml:1]: you can remove file

File: app/src/main/res/xml/data_extraction_rules.xml [hunk 8]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kkuan2011) [inline app/src/main/res/xml/data_extraction_rules.xml:1]: you can remove file

**Review by kkuan2011 — CHANGES_REQUESTED** [review]

File: settings.gradle [hunk 9]
```diff
@@ -0,0 +1,16 @@
+pluginManagement {
```
Comment (kkuan2011) [inline settings.gradle:1]: needs license, you can see examples here: https://github.com/android/compose-samples/find/main

**Review by kkuan2011 — CHANGES_REQUESTED** [review]

**Review by kkuan2011 — APPROVED** [review]

**Review by johnshea — APPROVED** [review]
lgtm

**Pull request merged**